lib/mayaUsd/resources/scripts/mayaUsdUtils.py

```diff
@@ -71,3 +75,80 @@ def isPulledMayaReference(dagPath):
 
     _, _, _, prim = getPulledInfo(dagPath)
     return prim and prim.GetTypeName() == 'MayaReference'
+
+
+def getMonoFormatFileFilterLabels(includeCompressed = True):
+    """
+    Returns a list of file-format labels for individual USD file formats.
+    This list is not directly usable in the dialog command, they would need
+    to be join with ';;'. Call getUSDDialogFileFilters() instead as it does
+    the joining.
+    """
+    if includeCompressed:
+        labelAndFilters = [
+            ("kUsdASCIIFiles", "*.usda"),
+            ("kUsdBinaryFiles", "*.usdc"),
+            ("kUsdCompressedFiles", "*.usdz"),
+        ]
+    else:
+        labelAndFilters = [
+            ("kUsdASCIIFiles", "*.usda"),
+            ("kUsdBinaryFiles", "*.usdc"),
+        ]
+    localizedLabels = [getMayaUsdLibString(labelKey) + ' ' + filter for labelKey, filter in labelAndFilters]
+    return localizedLabels
+
+
+def getMultiFormatsFileFilterLabels(includeCompressed = True):
+    """
+    Returns a list of file-format labels for multi-formats USD file formats.
+    This list is not directly usable in the dialog command, they would need
+    to be join with ';;'. Call getUSDDialogFileFilters() instead as it does
+    the joining.
+    """
+    if includeCompressed:
+        labelAndFilters = [
+            ("kAllUsdFiles", "(*.usd *.usda *.usdc *.usdz)"),
+            ("kUsdFiles", "*.usd"),
+        ]
+    else:
+        labelAndFilters = [
+            ("kAllUsdFiles", "(*.usd *.usda *.usdc)"),
+            ("kUsdFiles", "*.usd"),
+        ]
```

Comment:
Could we also do something to avoid duplication here? 
```suggestion
        labelAndFilters = [
            ("kAllUsdFiles", "(*.usd *.usda *.usdc{compressed})".format(compressed = " *.usdz" if includeCompressed else '')),
            ("kUsdFiles", "*.usd"),
        ]
```